umbraco-cms/fundamentals/backoffice/login.md

```diff
@@ -88,7 +94,7 @@ The `LoginBackgroundImage` and `LoginLogoImage` are referenced from the `/wwwroo
 
 To affect the styling of the images add a folder to the `/App_Plugins` folder containing a `CSS` folder, containing a CSS file, and a package.manifest file. An example structure might be:
 
-```
+```package.manifest
```

Comment:
Not sure if "package.manifest" is a syntax? 🤔

Reply:
No its not,

just read our docs and it should be JSON, so i have changed it :)